angular.json

```diff
@@ -23,10 +23,16 @@
               "src/assets"
             ],
             "styles": [
-              "node_modules/normalize.css/normalize.css",
-              "node_modules/material-design-icons/iconfont/material-icons.css",
-              "src/styles/main.css"
+              { "bundleName": "vendor-styles", "input": "node_modules/normalize.css/normalize.css" },
```

Comment:
Using bundle name, the styles are grouped, and in this way we can know how big are specific bundles:


### ng build

```
Date: 2018-07-23T09:11:31.069Z
Hash: 29d4baf6ed8368e9f003
Time: 7172ms
chunk {main} main.js, main.js.map (main) 22.2 kB [initial] [rendered]
chunk {main-styles} main-styles.js, main-styles.js.map (main-styles) 16 kB [initial] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 227 kB [initial] [rendered]
chunk {print-styles} print-styles.js, print-styles.js.map (print-styles) 16.9 kB [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 5.22 kB [entry] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 2.74 MB [initial] [rendered]
chunk {vendor-styles} vendor-styles.js, vendor-styles.js.map (vendor-styles) 25.6 kB [initial] [rendered]
```

### ng build --prod
```
Date: 2018-07-23T09:11:41.372Z
Hash: 5de8f59dddb7be2709e3
Time: 6080ms
chunk {0} runtime.a66f828dca56eeb90e02.js (runtime) 1.05 kB [entry] [rendered]
chunk {1} print-styles.88840b3438693b64d71f.css (print-styles) 635 bytes [initial] [rendered]
chunk {2} main-styles.13112db901a6ead70e56.css (main-styles) 44 bytes [initial] [rendered]
chunk {3} vendor-styles.fa4aee3cdfdc985fb226.css (vendor-styles) 2.57 kB [initial] [rendered]
chunk {4} polyfills.4064b73d3d36a95e5183.js (polyfills) 59.6 kB [initial] [rendered]
chunk {5} main.027c41c9e7aa896fa56e.js (main) 158 kB [initial] [rendered]
```